pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -10715,6 +10715,63 @@
 
   language_label: "Language"
 
+  hoc_homepage:
+    what_is_hoc:
+      heading: "What is the Hour of Code?"
+      desc: "The Hour of Code is worldwide movement that aims to introduce millions of students to computer science through one-hour coding activities. Through Hour of Code, we aim to demystify coding and show that anyone can learn the basics, inspiring future interest in computer science."
+    host:
+      heading: "Host an Hour of Code"
+      desc: "Your class or school can join millions around the world doing the Hour of Code! Registration for the annual Computer Science Education Week celebration begins in September. Though, Hour of Code is available year-round. Register today!"
```

Comment:
This "Though" feels very clunky to me, but I understand if that's the verbiage Marketing wants!

Reply:
Let's leave this one as-is!